docs/src/rsh/errors/index.md

```diff
@@ -2236,7 +2248,7 @@ a `{!rsh} Participant` or `{!rsh} ParticipantClass`.
 For example, the program below erroneously uses `{!rsh} Anybody.publish()` without declaring any `{!rsh} Participant` or `{!rsh} ParticipantClass`:
 
 ```reach
-load: /hs/t/n/Err_No_Participants.rsh
+load: ./hs/t/n/Err_No_Participants.rsh
```

Comment:
Doesn't this need to be changed?

Reply:
Maybe you need to rebase?

Reply:
Fixing it caused issues with merging for Hamir(who already had a fix), so I switched it back.

Reply:
Right now, if we were to merge yours, then it would be wrong

Reply:
Fixed it again